### PR TITLE
chore(jenkins-dcp): Fix dd + etlMapping + gitops.json

### PR DESCRIPTION
**Pull request by themarcelor**
In order to make this environment compatible with the upcoming PFB Export E2E test, we need to update the artifacts related to the Explorer page.

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# jenkins-dcp.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [data_type] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [data_format] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [data_format] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# jenkins-dcp.planx-pla.net/manifest.json
## :warning: Services on branch
- arborist
- audit-service
- awshelper
- fence
- indexd
- mariner
- ws-storage
- peregrine
- pidgin
- portal
- revproxy
- sheepdog
- spark
- tube
- ssjdispatcher
- manifestservice
- wts
- guppy
- sower
- hatchery
- metadata
- dashboard
- requestor

**Comment by PlanXCyborg** [comment]
# jenkins-dcp.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [data_type] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [data_format] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [data_format] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# jenkins-dcp.planx-pla.net/manifest.json
## :warning: Services on branch
- arborist
- audit-service
- awshelper
- fence
- indexd
- mariner
- ws-storage
- peregrine
- pidgin
- portal
- revproxy
- sheepdog
- spark
- tube
- ssjdispatcher
- manifestservice
- wts
- guppy
- sower
- hatchery
- metadata
- dashboard
- requestor

**Comment by PlanXCyborg** [comment]
# jenkins-dcp.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [data_type] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [data_format] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [data_format] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# jenkins-dcp.planx-pla.net/manifest.json
## :warning: Services on branch
- arborist
- audit-service
- awshelper
- fence
- indexd
- mariner
- ws-storage
- peregrine
- pidgin
- portal
- revproxy
- sheepdog
- spark
- tube
- ssjdispatcher
- manifestservice
- wts
- guppy
- sower
- hatchery
- metadata
- dashboard
- requestor

File: jenkins-dcp.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -290,56 +337,445 @@
     ],
     "arrangerConfig": {
```
Comment (mfshao) [inline jenkins-dcp.planx-pla.net/portal/gitops.json:338]: you might consider remving this guy, `arranger` has gone, and the portal config is already insanely long

**Review by mfshao — APPROVED** [review]

File: jenkins-dcp.planx-pla.net/portal/gitops.json (hunk 1 above)
Comment (themarcelor) [reply jenkins-dcp.planx-pla.net/portal/gitops.json:338]: ok, lemme get rid of it. 👍🏼 Thanks!

**Comment by PlanXCyborg** [comment]
# jenkins-dcp.planx-pla.net
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [data_type] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [data_format] in explorerConfig.filters.tabs.fields not found in etlMapping
- Field error: Field [data_format] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# jenkins-dcp.planx-pla.net/manifest.json
## :warning: Services on branch
- arborist
- audit-service
- awshelper
- fence
- indexd
- mariner
- ws-storage
- peregrine
- pidgin
- portal
- revproxy
- sheepdog
- spark
- tube
- ssjdispatcher
- manifestservice
- wts
- guppy
- sower
- hatchery
- metadata
- dashboard
- requestor

**Pull request merged**